ros2plugin/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2plugin</name>
+  <version>0.7.4</version>
+  <description>
+    The plugin command for ROS 2 command line tools.
+  </description>
+  <maintainer email="michel@ekumenlabs.com">Jeremie Deray</maintainer>
+  <license>Apache License 2.0</license>
+
+  <exec_depend>ament_index_python</exec_depend>
+  <!-- <exec_depend>composition_interfaces</exec_depend> -->
```

Comment:
Please remove comments like this if they are not containing helpful documentation.

Reply:
Cleaned up.